website/docs/r/fabric_wide_settings.html.markdown

```diff
@@ -46,11 +52,17 @@ Users can use the resource of type `aci_fabric_wide_settings` to change the conf
 * `description` - (Optional) Description of object Fabric-Wide Settings Policy.
 * `name_alias` - (Optional) Name alias of object Fabric-Wide Settings Policy.
 * `disable_ep_dampening` - (Optional) Disable Ep Dampening knob of object Fabric-Wide Settings Policy. Allowed values are "yes" and "no".
+* `domain_validation` - (Optional) Validate that static path is added but no domain is associated to an EPG.
```

Comment:
Validate that the correct physical domain is added before associating a new static path to an EPG.

Reply:
ok